source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -35,13 +42,22 @@ partitions = this would theoretically provide you up to: ``~896,000
 PUT/POST/DELETE TPS and 1,408,000`` GET TPS if requests are distributed evenly
 across the partitions).
 
+Configuration
+^^^^^^^^^^^^^
+
 .. code-block:: none
 
   plugin {
-    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
-    obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_index_fs = compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
   }
 
+.. Note::
+  The scheme ``aws-s3`` was newly introduced if your version does not support
+  it yet stick to ``s3``
```

Comment:
In general I don't like text like this because it quickly becomes out of date.  For example, in 6 months "newly" will feel a bit odd.  In a year, it'll be outright wrong :)

How about:

If the version you are using does not support the ``aws-s3`` scheme, try using the more generic ``s3`` scheme.